### ISSUE TITLE
Notes without a tag are excluded from search

### ISSUE THREAD

**Issue by julianmenzler** [bjrnt/alfred-bear#1]
I found this by accident. I don't think this is expected behavior, is it?

**Comment by bjrnt** [comment]
Hi Julian, 

Thanks for the bug report!

Could you try using the newly released version? I largely changed how searches are done with Bear. I've tried a few searches myself and found that untagged notes also appear as expected.

**Comment by bjrnt** [comment]
I'm going to close this due to inactivity, feel free to open another issue if there's anything I can help with!

**Comment by julianmenzler** [comment]
Hi Björn,
I just checked out your new version. Unfortunately it does work:
<img width="691" alt="image" src="https://user-images.githubusercontent.com/9380917/67845371-abf3d900-faff-11e9-95f7-41ba0ed27182.png">
It keeps loading and nothing happens.

Another thing I noticed: I hit the wrong repo. I was using a [different Go-based Alfred-Workflow](https://github.com/drgrib/alfred-bear). Sorry for the confusion.

Have a nice week!
